dsp-shared/src/test/scala/dsp/valueobjects/UserSpec.scala

```diff
@@ -5,10 +5,11 @@
 
 package dsp.valueobjects
 
+import dsp.errors.BadRequestException
```

Comment:
yes, quite a lot, since we didn't touch the user stuff, only LanguageCode. But actually, now that LanguageCode does not live with the user anymore, it shouldn't be tested here either.